test cases/common/13 pch/meson.build

```diff
@@ -1,6 +1,16 @@
-project('pch test', 'c', 'cpp',
+project('pch test', 'c',
   meson_version: '>= 0.46.0')
 
+if add_languages('cpp', required: false)
+  cpp = meson.get_compiler('cpp')
+else
+  error('MESON_SKIP_TEST No C++ compiler available.')
+endif
+
+if host_machine.system() == 'windows' and fcid == 'pgi'
```

Comment:
on this test, why don't you check for the cpp compiler and skip the c++ specific parts instead of skipping the whole thing?

Also, fcid is undefined in this function.

Reply:
resolved  eb6b99ca